### PR TITLE
Added related searches in case related questions don't exist for a pr…

### PR DESCRIPTION
**Pull request by LoganHoogendijk**
…ompt

### CODE REVIEW

File: API-testing/RelatedQuestions.py [hunk 1]
```diff
@@ -18,18 +18,39 @@ def __init__(self):
             "api_key": os.getenv("secret-api-key")
         }
 
-    def get_related_questions(self, query: string):
-        self.params["q"] = query
+    def get_related_questions(self, prompt: string):
+        self.params["q"] = prompt
```
Comment (SamEThibault) [inline API-testing/RelatedQuestions.py:22]: type hinting, nice!

**Review by SamEThibault — APPROVED** [review]

File: API-testing/RelatedQuestions.py (hunk 1 above)
Comment (LoganHoogendijk) [reply API-testing/RelatedQuestions.py:22]: Thanks! :D

File: API-testing/RelatedQuestions.py [hunk 2]
```diff
@@ -18,18 +18,39 @@ def __init__(self):
...
+            RS_query = [sub['query']
+                        for sub in related_searches]
+            return RS_query
+        else:
+            return None
 
 
 if __name__ == "__main__":
     questions = RelatedQuestions()
-    result = questions.get_related_questions("How to deal with depression")
```
Comment (SamEThibault) [inline API-testing/RelatedQuestions.py:34]: ??

Comment (LoganHoogendijk) [reply API-testing/RelatedQuestions.py:34]: I was just testing...

**Pull request merged**